### DISCUSSION TITLE
recursos

### DISCUSSION THREAD

**Discussion by crizoth** [UNIVALI-LITE/Portugol-Studio#944] [General]
Ola!
Uso sua plataforma para ensinar logica de programação ha 1 ano e meio, e seria interessante se houvesse uma forma de importar outro arquivo como .por, e utiliza-lo, como uma classe, se houver alguma forma de fazer isso ou algo parecido pra que possa iniciar o conceito de orientação a objetos com os alunos gostaria de saber, há uma biblioteca objetos , porém nao acho conteudo suficiente para compreende-la bem

**Comment by AdsonEsteves** [comment]
@crizoth 
Obrigado pela  sugestão.
Sim, já pensamos sobre essa possibilidade. Porém uma mudança pra orientação a objeto no momento ia depender de uma mudança relativamente grande de como o núcleo funciona. E no momento não temos como alocar recursos para implementa-la. 
Já também pensamos na implementacao de structs que devem ser mais fáceis de implementar, mas mesmo assim, levaria um bom tempo.

Então no momento não há como fazer isso. Há a biblioteca de objetos, mas são apenas objetos Json, portanto não funcionaria aplicar métodos como a orientacao a objetos pede.
Acredito que no médio-curto prazo também será difícil ter, já que não estamos com muitos recursos para isso. Sinto muito.
Mas obrigado por utilizar o Portugol Studio. :)

**Comment by crizoth** [comment]
muito obrigado  pela resposta

Em qui, 10 de jun de 2021 17:57, Adsu ***@***.***> escreveu:
